translations/pt_BR/i18n.ts

```diff
@@ -526,21 +526,21 @@ var t_pt_BR: TalkyardTranslations = t = {
     TooltipQuestSolved: "Isto é uma pergunta respondida",
     TooltipQuestUnsolved: "Isto é uma pergunta não respondida",
 
-    StatusDone: "Done",  // MISSING
+    StatusDone: "Concluído",
     TooltipProblFixed: "Isto foi consertado",
     TooltipDone: "Isto foi concluído",
 
-    StatusStarted: "Started",  // MISSING
+    StatusStarted: "Iniciado",
     TooltipFixing: "No momento estamos consertando isto",
     TooltipImplementing: "No momento estamos implementando isto",
 
-    StatusPlanned: "Planned",  // MISSING
+    StatusPlanned: "Planejado",
     TooltipProblPlanned: "Estamos planejando consertar isto",
     TooltipIdeaPlanned: "Estamos planejando implementar isto",
 
-    StatusNew: "New",  // MISSING
-    StatusNewDtl: "New topic, under discussion",  // MISSING
-    TooltipUnsProbl: "Isto é um problema não resolvido",
+    StatusNew: "Novo",
+    StatusNewDtl: "Novo tópico, em discussão",
+    TooltipUnsProbl: "Isso é um problema sem solução",
```

Comment:
Are bot `Isto é um problema não resolvido` and `Novo tópico, em discussão` Brazilian Portuguese?  (none of them Portuguese Portuguese?)

I'm curious about what's the difference :- )  Anyway, `sem solução` is a bit shorter, that's nice